docs/development.md

```diff
@@ -0,0 +1,18 @@
+# Development Notes
+
+## Arrow Functions
+This projet heavily prefers the use of the `function` keyword over the use of arrow functions in most cases.
```

Comment:
Link to MDN in this sentence:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions